released/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.5.3/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+maintainer: "Reynald Affeldt <reynald.affeldt@aist.go.jp>"
+
+homepage: "https://github.com/math-comp/analysis"
+dev-repo: "git+https://github.com/math-comp/analysis.git"
+bug-reports: "https://github.com/math-comp/analysis/issues"
+license: "CECILL-C"
+
+synopsis: "An analysis library for mathematical components"
+description: """
+This repository contains an experimental library for real analysis for
+the Coq proof-assistant and using the Mathematical Components library."""
+
+build: [make "-j%{jobs}%"]
+install: [make "install"]
+depends: [
+  "coq" { (>= "8.14" & < "8.16~") | (= "dev") }
```

Comment:
```suggestion
  "coq" { (>= "8.14" & < "8.17~") | (= "dev") }
```
To support the soon-to-be-released Coq 8.16.0.

Reply:
I haven't tested it yet though.

Reply:
@affeldt-aist but do you mind that we have this for now to *enable* testing? If there is some problem, I can edit the package again.

Reply:
I do not mind.

Reply:
FYI, it appears that an installation with Coq 8.16+rc1, OCaml 4.14.0, HB 1.3.0, and dev versions of MathComp succeeded. @gares